Lib/pdb.py

```diff
@@ -998,6 +984,13 @@ def _pdbcmd_print_frame_status(self, arg):
         self.print_stack_trace(0)
         self._show_display()
 
+    def _pdbcmd_silent_frame_status(self, arg):
```

Comment:
should this be 
```suggestion
    def _pdbcmd_pop_silent_frame_status(self, arg):
```
?

Reply:
”silent“ here is a verb. Maybe there are better words for it?

Reply:
did you mean silence?

Reply:
Yes!

Reply:
`silent` is used as a command in ``commands``, which is probably copied from `gdb`. However, a verb here makes more sense so I'll change the function name to slience.

Reply:
"set to silent" also works

Reply:
I changed this to `silence` and I think it worked well :)

Anything else I should do for this PR?